### PR TITLE
Add files via upload

### PR DESCRIPTION
**Pull request by Sampannas**

### CODE REVIEW

**Review by LucasHT22 — COMMENTED** [review]
Thanks for PR!

File: UNTITLED (1).js [hunk 1]
```diff
@@ -0,0 +1,176 @@
+/*
+@author Dhruv
```
Comment (LucasHT22) [inline UNTITLED (1).js:2]: 2 author?

**Comment by LucasHT22** [comment]
good puzzles! could you add another 2 levels?

File: UNTITLED (1).js [hunk 2]
```diff
@@ -0,0 +1,244 @@
...
+4444444444444444
+4444444444444444
+4444444444444444
+4444444444444444
+4444444444444444
+4444444444444444
+4444444444444444
+4444444444444444`]
+)
+let level = 3;
```
Comment (LucasHT22) [inline UNTITLED (1).js:60]: good new levels!
to start correctly:
`let level = 0;`

**Comment by LucasHT22** [comment]
whats the name of the game?

**Comment by Sampannas** [comment]
Hoe about just "The Maze"?

File: UNTITLED (1).js (hunk 2 above)
Comment (Sampannas) [reply UNTITLED (1).js:60]: Oh yea, I forgot to change it back to 0.

**Comment by LucasHT22** [comment]
hmmm we already have a lot of maze in the gallery, how about Escape Room?

**Comment by Sampannas** [comment]
Yea, that sounds much better.

**Comment by LucasHT22** [comment]
did I make the updates approved by you, before merging, 2 author?

**Comment by Sampannas** [comment]
yea? I don't know

**Comment by LucasHT22** [comment]
did you make this game with Dhruv?
image

**Comment by Sampannas** [comment]
oh, yea. We did it together.

**Comment by LucasHT22** [comment]
Then leave it on one line, 
`@author: Sampanna and Dhruv`

**Comment by Sampannas** [comment]
ok!

**Comment by LucasHT22** [comment]
so make a commit with that change and I merge :)

**Review by LucasHT22 — APPROVED** [review]
Congrats!
If you're a teen, you can get your sprig here! https://sprig-order.hackclub.dev/

**Pull request merged**